libmscore/rendermidi.cpp

```diff
@@ -17,6 +17,7 @@
 
 #include <set>
 
+#include "assert.h"
```

Comment:
Yes you're right; we no longer need assert.h.  Thanks.  unfortunately the compiler does not warm me if I include an unnecessary header file.